stack.cabal

```diff
@@ -268,6 +268,7 @@ library
                    , store-core >= 0.4 && < 0.5
                    , annotated-wl-pprint
                    , file-embed >= 0.0.10
+  build-tools:       hsc2hs >= 0.68
```

Comment:
Is this even correct? It doesn't seem to help the appveyor problem, but maybe it's supposed to be there anyway?

I've never really used `build-tools` before, and I also can't tell if `hsc2hs` is part of GHC and thus doesn't need to be listed here or what.

Reply:
Hmm, could be that hsc2hs needs to be added to stackage.  For now, adding `stack install hsc2hs` to the travis script should fix the issue.  Adding hsc2hs to extra-deps might help, as it would specify a version to install, not 100% sure if that works, it used to not, but that may have been fixed.

Reply:
Oh, nice catch, it isn't in stackage. Appveyor seems to be the only one that doesn't like it. Huh, I'm getting other travis CI errors that look like actual code problems. Wacky that I'm not getting it locally though, I'll have to look into that.